html/browsers/the-window-object/close-method.window.js

```diff
@@ -0,0 +1,30 @@
+async_test(t => {
+  const openee = window.open();
+  assert_false(openee.closed);
+  openee.close();
+  assert_true(openee.closed);
+  assert_equals(openee.opener, self);
+  t.step_timeout(() => {
+    assert_equals(openee.opener, null);
+    t.done();
+  }, 250);
```

Comment:
Just realized: could you make these tests reliable by using the pagehide or unload event, plus a setTimeout(,0)? Seems like it'd be a tighter test too.